tox_pipenv/plugin.py

```diff
@@ -94,92 +76,46 @@ def tox_testenv_install_deps(venv, action):
         deps = venv.get_resolved_dependencies()
     basepath = venv.path.dirpath()
     basepath.ensure(dir=1)
-    pipfile_path = _clone_pipfile(venv)
-    args = [sys.executable, "-m", "pipenv", "install", "--dev"]
+    pipfile_path, pipfile_lock_path = _clone_pipfile(venv)
+
+    install_args = _env_install_args()
+    if install_args is None:
+        if pipfile_lock_path is not None:
+            # project provided a lock file, so sync deps
+            install_args = ["sync", "--dev"]
+        else:
+            install_args = ["install", "--dev"]
+
+    args = [sys.executable, "-m", "pipenv"] + install_args
```

Comment:
Should we use the `python` from the venv?
```suggestion
    args = [sys.executable, "-m", "pipenv", "install", "--python", venv.path.join("bin", "python")] + install_args
```
or something like that... (This relies on my other feedback about not accepting the command in `TOX_PIPENV_INSTALL_ARGS`.)

Reply:
I'm not sure it matters when pipenv isn't creating the venv.